sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -364,8 +367,16 @@ static void _print_reply(_ping_data_t *data, gnrc_pktsnip_t *icmpv6,
             data->num_recv++;
             dupmsg += 7;
         }
-        printf("%u bytes from %s: icmp_seq=%u ttl=%u", (unsigned)icmpv6->size,
-               from_str, recv_seq, hoplimit);
+        if (GNRC_NETIF_NUMOF == 1 || if_pid == 0) {
```

Comment:
You just need to add a `ipv6_addr_is_link_local(from)` here to have global addresses not be outputted with an interface.